src/types/intentions.ts

```diff
@@ -1,8 +1,20 @@
-const AVIARY_INTENTIONS = {
+const CORE_AVIARY_INTENTIONS = {
   primary: "primary",
   system: "system",
   danger: "danger",
```

Comment:
I haven't counted, but I feel like we should switch warning and danger here, and have warning be in the core intentions instead. I believe we're removing most of the danger intentions because it's too strong